packages/salesforcedx-vscode-core/package.json

```diff
@@ -155,17 +155,27 @@
         {
           "command": "sfdx.force.source.retrieve",
           "when":
-            "sfdx:project_opened && sfdx:default_username_has_no_change_tracking"
+            "sfdx:project_opened && sfdx:default_username_has_no_change_tracking && resourceLangId != 'forcesourcemanifest'"
```

Comment:
We should also add `&& resourceFilename != manifest` so that these commands (retrieve, deploy, delete) are not showed when right-clicking the manifest folder.

Reply:
I agree but right now, all of those other commands should be taken care of with the other work item. I've done the minimum to prevent them from being on the menu when the *Manifest* commands are active.